go.mod

```diff
@@ -6,6 +6,9 @@ replace (
 	// Use nested module.
 	github.com/siderolabs/talos/pkg/machinery => ./pkg/machinery
 
+	// see https://github.com/vmware-tanzu/sonobuoy/pull/1933
+	github.com/vmware-tanzu/sonobuoy => github.com/smira/sonobuoy v0.0.0-20230925141431-e9307f0a884d
```

Comment:
i guess we get this in now and wait for upstream?

Reply:
Yes, otherwise it would waste 500% of the CPU by the end of the test